ocs_ci/ocs/fiojob.py

```diff
@@ -288,6 +288,38 @@ def delete_fio_data(fio_job_file, delete_check_func):
         raise Exception(error_msg)
 
 
+def get_sc_name(fixture_name):
```

Comment:
We have one similar function defined [here](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/helpers/helpers.py#L457) to get the default_storage_class

Reply:
Hmm, the code related to storage class management would benefit from cleanup and it's own dedicated module (https://github.com/red-hat-storage/ocs-ci/issues/3688). I'm keeping  get_sc_name as it is for now, but when #3688 is addressed, it should be fixed.